### ISSUE TITLE
[Question] Where should I put the art assets?

### ISSUE THREAD

**Issue by birdkingz** [pixijs-userland/spine#244]
Hi, I'm new to pixi and spine. I don't know where to put the art assets in my project.

The spine examples that I found [here](https://pixijs.io/examples/?v=v4.7.3#/spine/dragon.js) just load the .json file.

```
// load spine data
PIXI.loader
    .add('dragon', 'required/assets/spine/dragon.json')
    .load(onAssetsLoaded);
``` 

So I wonder where should I put all the .png files?
Why the sample doesn't need to load individual textures before initiating the spine animation?

**Comment by ivanpopelyshev** [comment]
this code loads them from `required/assets/spine/` relative folder. You can change it, or change `PIXI.loader.baseUrl` if you want to separate the folder and filenames (if you have 10 adds and dont like CTRL+C)

**Comment by ivanpopelyshev** [comment]
Take the code from it, it works. You are doing something basic wrong. I need more information.

**Comment by birdkingz** [comment]
Do you think is that possible something wrong with the asset files?
Here are the [link](https://drive.google.com/open?id=1uU_qlkAc4HaXDclmFQrsntAVv5Ayyw6X) to download the assets that I'm using

**Comment by ivanpopelyshev** [comment]
sorry, this case is to mysterious, my telepathy is not working on it. Give me full zip file and we'll see.

**Comment by birdkingz** [comment]
[Here](https://drive.google.com/file/d/16o9toE8LqmO7BBZW5YDLGag51zQbeXs-/view?usp=sharing) you go

You need to run `npm install` (in terminal) to install the required modules before running `npm start` (in terminal)

**Comment by ivanpopelyshev** [comment]
Ok, I'll look at it after work.

**Comment by djmisterjon** [comment]
@birdkingz 
hum what about your version ?
Example my side i have this.
`"skeleton": { "hash": "I5g2nJYqMDHwtlJeytbv5Okk57k", "spine": "3.6.52", "width": 130.22, "height": 232.18, "images": "./images/" },`
you side you have this.
`"skeleton": { "hash": "9MKo2cmJTDc3IPV4B3LRJxWbl04", "spine": "3.4.02", "width": 897, "height": 716.36, "images": "/images/game/dragon" },`

**I have already get this error,** I no longer remember what was the reason, and how I corrected it.
I'm really not sure, but I think I had that kind of error when I turned off webGL to use only canvas mode.
But I'm really not sure.
Sorry for not being able to help you more, maybe ivan will be luckier.
